client/src/app/site/base/base-filter.service/base-filter-list.service.ts

```diff
@@ -293,7 +296,8 @@ export abstract class BaseFilterListService<V extends BaseViewModel> implements
                                               condition: child.id
                                           }))
                                         : undefined
-                            }));
+                            }))
+                            .sort(sortFn);
```

Comment:
No need to declare the sort function somewhere else. 

```suggestion
                            .sort((a, b) => a.label.trim().localeCompare(b.label.trim()));
```